dockerfiles/cli/cli.sh

```diff
@@ -687,6 +687,7 @@ generate_configuration_with_puppet() {
                   -e "REGISTRY_ENV_FILE=${REGISTRY_ENV_FILE}" \
                   -e "POSTGRES_ENV_FILE=${POSTGRES_ENV_FILE}" \
                   -e "CODENVY_ENV_FILE=${CODENVY_ENV_FILE}" \
+                  -e "ENABLE_JPDA_SUSPEND=${ENABLE_JPDA_SUSPEND:-false}" \
```

Comment:
it looks strange that we need to set extra env value there It's not in codenvy.env ?